app/workers/moderator/sink_articles_worker.rb

```diff
@@ -8,10 +8,7 @@ def perform(user_id)
       user = User.find_by(id: user_id)
       return unless user
 
-      articles = Article.where(user: user)
-      reactions = Reaction.where(reactable: articles)
-      new_score = reactions.sum(:points) + Reaction.where(reactable: user).sum(:points)
-      articles.update_all(score: new_score)
+      Article.where(user: user).each(&:update_score)
```

Comment:
We have 28 users with > 1000 articles (though the top one has 20,277 drafts and 0 published articles) and 56 with over 500 articles. So maybe we can change this to `find_each` with a  batch size of 100 or so, just in case.